libopenage/engine/engine.cpp

```diff
@@ -16,14 +16,20 @@ namespace openage::engine {
 Engine::Engine(mode mode,
                const util::Path &root_dir,
                const std::vector<std::string> &mods,
-               bool debug_graphics) :
+               bool debug_graphics,
+			   int wWidth,
+			   int wHeight) :
```

Comment:
We use snake case for variable names, so `window_width` or just `width` are better.

Reply:
Also, you can use `size_t` instead of `int` for the type because the width/height shouldn't be negative.